godot-codegen/src/special_cases.rs

```diff
@@ -47,9 +47,15 @@ pub(crate) fn is_class_deleted(class_name: &TyName) -> bool {
 
     match class_name.godot_ty.as_str() {
         // Hardcoded cases that are not accessible.
-        | "JavaClassWrapper" // only on Android.
-        | "JavaScriptBridge" // only on WASM.
-        | "ThemeDB" // lazily loaded; TODO enable this.
+        // Only on Android.
+        | "JavaClassWrapper"
+        | "JNISingleton"
+        | "JavaClass"
+        // Only on WASM.
+        | "JavaScriptBridge"
+        | "JavaScriptObject"
```

Comment:
Oh nice, how did you find those classes? It looks like their presence didn't trigger an error (even now with full-ci checking everything)?

Reply:
I made a script to write a file that made a class inheriting from every single class in the gdextension api. some classes crashed on startup because i assume they're not meant to be extended (like `AudioServer`), but these threw an error because they werent available.